main.go

```diff
@@ -68,13 +68,22 @@ var (
 )
 
 func probeHandler(w http.ResponseWriter, r *http.Request, c *config.Config, logger log.Logger, rh *resultHistory) {
+	probeModuleUnknownGauge := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "probe_module_unknown",
+		Help: "Displays whether or not probe module is unknown",
+	})
+	registry := prometheus.NewRegistry()
+	registry.MustRegister(probeModuleUnknownGauge)
+
 	moduleName := r.URL.Query().Get("module")
 	if moduleName == "" {
 		moduleName = "http_2xx"
 	}
 	module, ok := c.Modules[moduleName]
 	if !ok {
 		http.Error(w, fmt.Sprintf("Unknown module %q", moduleName), http.StatusBadRequest)
+		level.Error(logger).Log("msg", "Error ", "err", fmt.Sprintf("unknown module %s", moduleName))
```

Comment:
No need to spam the logs when someone has incorrect usage.

Reply:
Remove or switch to `debug`?

Reply:
I have switched to `debug`